packages/ui-components/src/components/labels-dropdown/labels-dropdown.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../assets/styles/globals';
+
+.selected-labels-container {
+	display: flex;
+	align-items: center;
+	height: 100%;
+	margin-left: $spacing;
```

Comment:
Is this correct?

Reply:
It is, I assume you made this comment based on the one below ;) I've assigned the correct value to the const :)